Update TGEX 2022 article (#7424)
ppy/osu-wiki: wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md

# Triton Gaming Expo 2022

!["Welcome to TGEX" pixel art banner from the main spreadsheet](img/spreadsheet-pixel-art.png)

> **Triton Gaming Expo 2022** (***TGEX 2022***) was an osu! BYOC LAN event hosted by [osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) in collaboration with *Triton Gaming*. It shared a name with its parent event, a gaming convention spanning many more games and activities. This event and tournament was osu! at UCSD's fourth appearance at TGEX.

Reviewer: ```suggestion
**Triton Gaming Expo 2022** (***TGEX 2022***) was an osu! BYOC LAN event hosted by [osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) in collaboration with *Triton Gaming*. It shared its name with its parent event, a gaming convention spanning many more games and activities. This event and tournament was osu! at UCSD's fourth appearance at TGEX.
```

## TGEX

![TGEX banner art](img/tgex-banner.png)

TGEX 2022 took place on Sunday, May 29th at [Price Center](https://en.wikipedia.org/wiki/Price_Center) at the [University of California, San Diego](https://en.wikipedia.org/wiki/University_of_California,_San_Diego) (*UCSD*). The main floor was open from 12:00 p.m. to 10:00 p.m., but each event inside had its own schedule. The osu! LAN ran from around 11:15 a.m. to 9:00 p.m. in Earl Warren College Room.

Aside from the osu! event, TGEX hosted tournaments for many other games, panels with game industry experts, an artist alley, cosplay gathering, keyboard showcase, and more. More details can be found in the [Triton Gaming Discord server](https://discord.gg/tritongaming).

## Links

- [Main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit)
- [Challonge bracket](https://challonge.com/yqvqkyfq)
- [Seeding results spreadsheet](https://docs.google.com/spreadsheets/d/1WZbENIjcmtDUcwqj-heN7MfDbsBZ8znvfprOP-Sve0g/edit)
- [osu! at UCSD Discord server](https://discord.gg/DqBVmv5Knx)
- [Triton Gaming Discord server](https://discord.gg/tritongaming)

> ## Format

Reviewer: ```suggestion
## Tournament format
```

> TGEX 2022 was a 1-on-1 double-elimination tournament with a seeding round before the bracket. More details and rules can be found in the ["Info & Rules" tab of the main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit#gid=1843073213).

Reviewer: ```suggestion
The tournament hosted at TGEX 2022 was a 1v1 double-elimination tournament with a seeding round before the bracket. More details and rules can be found in the ["Info & Rules" tab of the main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit#gid=1843073213).
```

> The tournament hosted at TGEX 2022 was a 1v1 double-elimination tournament with a seeding round before the bracket. More details and rules can be found in the ["Info & Rules" tab of the main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit#gid=1843073213).

Reviewer: also part of the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md#title) to specify the game mode

```suggestion
The tournament hosted at TGEX 2022 was a 1v1 double-elimination osu! tournament with a seeding round before the bracket. More details and rules can be found in the ["Info & Rules" tab of the main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit#gid=1843073213).
```

- Hidden
  1. [Wakeshima Kanon - killy killy JOKER (P A N) [eXtra]](https://osu.ppy.sh/beatmapsets/277901#osu/629665)
  2. [Amane - BOOZEHOUND (tieff) [Lesjuh]](https://osu.ppy.sh/beatmapsets/9995#osu/39549)
- HardRock
  1. [Rchetype - V Is For (Blue Dragon) [Insane]](https://osu.ppy.sh/beatmapsets/111914#osu/290919)
  2. [Nero's Day At Disneyland - No Money Down, Low Monthly Payments (grumd) [Insane]](https://osu.ppy.sh/beatmapsets/111825#osu/290733)
- DoubleTime
  1. [azusa - Check my soul (Sumisola) [Insane]](https://osu.ppy.sh/beatmapsets/45664#osu/142462)
  2. [Shoichiro Sakamoto - Eye of Aeon (KanaRin) [Insane]](https://osu.ppy.sh/beatmapsets/26528#osu/91139)
  3. [nano - Now or Never (tutuhaha) [Now]](https://osu.ppy.sh/beatmapsets/51755#osu/158498)
- ForceMod
  1. [HitoshizukuP x Yama - Crazy nighT (Sephibro) [Crazy]](https://osu.ppy.sh/beatmapsets/109401#osu/285549)
  2. [Kawada Mami - u/n (Reiji Maigo) [Insane]](https://osu.ppy.sh/beatmapsets/1142576#osu/2470326)
- Tiebreaker
  1. **[Uchu Conbini - Compass (Sing) [Space]](https://osu.ppy.sh/beatmapsets/1349881#osu/2794869)**

> ### Seeding

Reviewer: the actual seedings aren't here

```suggestion
### Seeding round
```

- HardRock
  1. [THE ORAL CIGARETTES - 5150 (Kagetsu) [Collab Extra]](https://osu.ppy.sh/beatmapsets/705655#osu/1532288)
  2. [HyuN - Guitian (ScubDomino) [Insane]](https://osu.ppy.sh/beatmapsets/891348#osu/1863256)
- DoubleTime
  1. [senya - Koyoi Index (jonathanlfj) [Lunatic]](https://osu.ppy.sh/beatmapsets/234455#osu/543814)
  2. [Tsunamaru - Gin no Fluch (HelloSCV) [Insane]](https://osu.ppy.sh/beatmapsets/85721#osu/235309)

## Match results

### Grand Finals

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **[Chzaron](https://osu.ppy.sh/users/11607790)** ![][flag_US] | **6** | 1 | ![][flag_US] [Kenny](https://osu.ppy.sh/users/1225459) | [#1](https://osu.ppy.sh/community/matches/101038923) |

### Finals

#### Winners bracket

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **[Chzaron](https://osu.ppy.sh/users/11607790)** ![][flag_US] | **5** | 2 | ![][flag_US] [-ether-](https://osu.ppy.sh/users/6131258) | [#1](https://osu.ppy.sh/community/matches/101037695) |

#### Losers bracket

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |

> | [-ether-](https://osu.ppy.sh/users/6131258) ![][flag_US] | -1 | **0** | ![][flag_US] **[Kenny](https://osu.ppy.sh/users/1225459)** | *Forfeit* |

Reviewer: ```suggestion
| [-ether-](https://osu.ppy.sh/users/6131258) ![][flag_US] | -1 | **0** | ![][flag_US] **[Kenny](https://osu.ppy.sh/users/1225459)** | *forfeit* |
```

Author: why not sentence case

Reviewer: because [tournament template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md#stage-1) and convention

Author: i feel like neither should matter if it just looks dumb but alright.

| **[Strider](https://osu.ppy.sh/users/4553852)** ![][flag_US] | **4** | 0 | ![][flag_US] [jms8719](https://osu.ppy.sh/users/2042565) | [#1](https://osu.ppy.sh/community/matches/101031762) |
| **[crikhard](https://osu.ppy.sh/users/7023000)** ![][flag_US] | **4** | 0 | ![][flag_US] [freddiiieeee](https://osu.ppy.sh/users/7112839) | [#1](https://osu.ppy.sh/community/matches/101031744) |
| **[-ether-](https://osu.ppy.sh/users/6131258)** ![][flag_US] | **4** | 3 | ![][flag_US] [leeeroy](https://osu.ppy.sh/users/12620812) | [#1](https://osu.ppy.sh/community/matches/101031752) |

#### Losers bracket

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| [freddiiieeee](https://osu.ppy.sh/users/7112839) ![][flag_US] | 0 | **4** | ![][flag_US] **[leeeroy](https://osu.ppy.sh/users/12620812)** | [#1](https://osu.ppy.sh/community/matches/101033421) |
| [jms8719](https://osu.ppy.sh/users/2042565) ![][flag_US] | 3 | **4** | ![][flag_US] **[Vysion](https://osu.ppy.sh/users/11594132)** | [#1](https://osu.ppy.sh/community/matches/101033430) |

> | [SeaAE](https://osu.ppy.sh/users/12354364) ![][flag_US] | -1 | **0** | ![][flag_US] **[jazzberry](https://osu.ppy.sh/users/11055678)** | *Forfeit* |

Reviewer: ```suggestion
| [SeaAE](https://osu.ppy.sh/users/12354364) ![][flag_US] | -1 | **0** | ![][flag_US] **[jazzberry](https://osu.ppy.sh/users/11055678)** | *forfeit* |
```

| **[dyndenial](https://osu.ppy.sh/users/16779673)** ![][flag_US] | **4** | 0 | ![][flag_US] [firenoo](https://osu.ppy.sh/users/10815041) | [#1](https://osu.ppy.sh/community/matches/101033452) |

### Round 1

#### Winners bracket

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **[firenoo](https://osu.ppy.sh/users/10815041)** ![][flag_US] | **4** | 0 | ![][flag_US] [atlae99](https://osu.ppy.sh/users/29787503) | [#1](https://osu.ppy.sh/community/matches/101030746) |

#### Losers bracket

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **[leeeroy](https://osu.ppy.sh/users/12620812)** ![][flag_US] | **4** | 0 | ![][flag_US] [atlae99](https://osu.ppy.sh/users/29787503) | [#1](https://osu.ppy.sh/community/matches/101032916) |

### Seeding

> Seeding was done online the day prior to the LAN. Every participant played every map in the seeding pool, and they were seeded by their average placement. Details can be found in the [seeding results spreadsheet](https://docs.google.com/spreadsheets/d/1WZbENIjcmtDUcwqj-heN7MfDbsBZ8znvfprOP-Sve0g/edit).

Reviewer: ```suggestion
The seeding was done online the day prior to the LAN. Every participant played every map in the seeding mappool, and they were seeded by their average placement. Details can be found in the [seeding results spreadsheet](https://docs.google.com/spreadsheets/d/1WZbENIjcmtDUcwqj-heN7MfDbsBZ8znvfprOP-Sve0g/edit).
```
